### PR TITLE
Create issue_template.md

### PR DESCRIPTION
**Pull request by hmarchadour**

### CODE REVIEW

**Review by sbegaudeau — CHANGES_REQUESTED** [review]
Could you use the same Git commit message pattern as used since the beginning of the project?

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+       - [ ] Linux
+       - [ ] macOS
+   - [ ] Cloud
+   - [ ] other: please specify
+
+### Actual behavior
+
+...
+
+### Steps to reproduce
```
Comment (pcdavid) [inline .github/ISSUE_TEMPLATE/issue_template.md:20]: I would put "Steps to reproduce" first.

"Here's what I did. I expected the software to do this, but it did that other thing instead which I think is wrong."

**Review by pcdavid — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+* [ ] **I have checked that this bug has not yet been reported by someone else.**
+* [ ] **I have checked that this bug appears on Chrome.**
+
+### Version
```
[the remark this answers is not in the document]
Comment (hmarchadour) [reply .github/ISSUE_TEMPLATE/issue_template.md:4]: You are right but we can expect to deploy/use a specific version soon does'nt it? I've add this field after a @mbats feedback in my previous PR with a bad ECA...

File: .github/ISSUE_TEMPLATE/issue_template.md (hunk 1 above)
Comment (hmarchadour) [reply .github/ISSUE_TEMPLATE/issue_template.md:20]: ==>
### Steps to reproduce

...

### Expected behavior

...

### Actual behavior

...

File: .github/ISSUE_TEMPLATE/issue_template.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (hmarchadour) [reply .github/ISSUE_TEMPLATE/issue_template.md:4]: https://github.com/eclipse-sirius/sirius-components/pull/121#discussion_r531082521

**Review by sbegaudeau — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+* [ ] **I have checked that this bug has not yet been reported by someone else**
+* [ ] **I have checked that this bug appears on Chrome**
+* [ ] **I have specify the version** :
```
Comment (pcdavid) [inline .github/ISSUE_TEMPLATE/issue_template.md:3]: Typo: "specified"

File: .github/ISSUE_TEMPLATE/issue_template.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- latest
+- other: please specify
+* [ ] **I have specify my environment** :
```
Comment (pcdavid) [inline .github/ISSUE_TEMPLATE/issue_template.md:6]: Typo: "specified"

**Review by pcdavid — CHANGES_REQUESTED** [review]

**Review by pcdavid — APPROVED** [review]

**Pull request merged**